api/config/cdr_config_perf.json

```diff
@@ -7,6 +7,14 @@
       "servicePerimeter": "accessPolicies/228353087260/servicePerimeters/terra_perf_aou_perf",
       "authDomainName": "all-of-us-registered-perf",
       "authDomainGroupEmail": "all-of-us-registered-perf@perf.test.firecloud.org"
+    },
+    {
+      "accessTierId": 2,
+      "shortName": "controlled",
+      "displayName": "Fake Controlled Tier For Perf",
```

Comment:
why it is `Fake`?

Reply:
Just so no one gets scared :)

This value is not important, and easy to change later if we need to.